File: PYTHON/Human.py
class Human:
    __name = ""
    __gender = ''

    def __init__(self, name = "", gender = ''):
        self.__name = name
        self.__gender = gender

    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name

    def get_gender(self):
        return self.__gender
    
    def set_gender(self, gender):
        self.__gender = gender

    def eat(self):
        print(self.__name, "is eating!")

    def sleep(self):
        print(self.__name, "is sleeping!")

File: JAVA/Human.java
public class Human{
    private String name;
    private char gender;

    public Human(){
        this.name = "";
        this.gender = '-';
    }

    public Human(String name, char gender){
        this.name = name;
        this.gender = gender;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public char getGender(){
        return this.gender;
    }

    public void setGender(char gender){
        this.gender = gender;
    }

    public void eat(){
        System.out.println(this.name + " is eating!");
    }

    public void sleep(){
        System.out.println(this.name + " is sleeping!");
    }
}